docs/01_nodeos/07_concepts/15_private_chain_access/index.md

```diff
@@ -0,0 +1,79 @@
+---
+content_title: Private Chain Access Control
+link_text: Private Chain Access Control
+---
+
+## Overview
+
+The *Private Access Control* or *Security Group* feature provides a two-layer security solution for blockchain administrators to control which participants can access and use a private EOSIO network. The first security layer enforces TLS connections in the peer-to-peer network protocol; only those participants with a properly signed certificate are able to establish a TLS connection and communicate securely with other peers. The second security layer, which is optional, brings the concept of a *privacy group* or *security group*; if enabled, only those participants in the group are allowed to access data from the private network. Therefore, the first layer controls “connection” access while the second layer manages “data” access. Participants are considered logical entities with a designated EOSIO name and may host multiple nodes, potentially deployed in [Block Vault](../../03_plugins/blockvault_client_plugin/index.md) mode.
```

Comment:
Is the full name of the feature *Private Access Control* or *Private Chain Access Control*? If the latter, then we have to change it in all instances. If the former, then no action required!

Reply:
Should be "Private Access Control", which is mainly applicable to private chains. Will update title.